finality-aleph/src/network/split.rs

```diff
@@ -8,6 +8,7 @@ use futures::channel::mpsc;
 use log::{debug, trace};
 use tokio::sync::Mutex;
 
+use super::SimpleNetwork;
```

Comment:
Small style thingy – we don't generally use `super` and I would keep it that way, especially since the line below is inconsistent with this.